composer/callbacks/speed_monitor.py

```diff
@@ -360,10 +361,18 @@ def batch_end(self, state: State, logger: Logger):
         # Log the time
         # `state.timestamp` excludes any time spent in evaluation
         train_wct = state.timestamp.total_wct.total_seconds()
+        secs_per_step = 0
+        if len(self.history_wct) > 1:
+            secs_per_step = self.history_wct[-1] - self.history_wct[-2]
+
         logger.log_metrics({
             'time/train': train_wct / self.divider,
             'time/val': self.total_eval_wct / self.divider,
             'time/total': (train_wct + self.total_eval_wct) / self.divider,
+            f'time_{self.time_unit}/train': train_wct / self.divider,
+            f'time_{self.time_unit}/val': self.total_eval_wct / self.divider,
+            f'time_{self.time_unit}/total': (train_wct + self.total_eval_wct) / self.divider,
```

Comment:
Let's just rename `time/train` -> `time/train/hours`? 

Changing log keys is a little annoying to end user... but duplicate logging is more annoying imo.

Reply:
@mvpatel2000 i was a bit worried this is a breaking change for any downstream pipelines that process metrics.

Reply:
Discussed offline, it's better to avoid overwhelming loggers